plugin/storage/es/dependencystore/storage.go

```diff
@@ -121,17 +122,12 @@ func buildTSQuery(endTs time.Time, lookback time.Duration) elastic.Query {
 	return elastic.NewRangeQuery("timestamp").Gte(endTs.Add(-lookback)).Lte(endTs)
 }
 
-func (s *DependencyStore) getIndices(ts time.Time, lookback time.Duration) []string {
+func getIndices(ts time.Time, lookback time.Duration) []string {
 	var indices []string
 	indices = append(indices, indexName(ts))
 	for lookback > 0 {
```

Comment:
didn't you have some logic in a different file that did this more carefully? ie even if ts - lookback falls on the same day as ts, we're still adding the index for the previous day,